test/polytopes.jl

```diff
@@ -3,20 +3,28 @@
     @test paramdim(Segment) == 1
     @test nvertices(Segment) == 2
 
+    s = Segment(Point(1.0,), Point(2.0,))
+    @test all(Point(x,) ∈ s for x in 1:0.01:2)
+    @test all(Point(x,) ∉ s for x in [-1.0, 0.0, 0.99, 2.1, 5.0, 10.0])
     s = Segment(P2(0,0), P2(1,1))
     @test minimum(s) == P2(0,0)
     @test maximum(s) == P2(1,1)
     @test extrema(s) == (P2(0,0), P2(1,1))
     @test isapprox(length(s), sqrt(T(2)))
     @test s(T(0)) == P2(0,0)
     @test s(T(1)) == P2(1,1)
-    @test all(p ∈ s for p in [P2(0.11, 0.11), P2(0.5, 0.5), P2(0.87, 0.87)])
-    @test all(p ∉ s for p in [P2(-0.1, -0.1), P2(1.1, 1.1), P2(1, 2)])
+    @test all(P2(x,x) ∈ s for x in 0:0.01:1)
```

Comment:
Can you please preserve the original version of the test? We try to leave tests untouched unless there is an actual change of behavior. The second modification is fine because you added a new point to the list.